lsp-mode.el

```diff
@@ -5,7 +5,7 @@
 ;; Author: Vibhav Pant, Fangrui Song, Ivan Yonchovski
 ;; Keywords: languages
 ;; Package-Requires: ((emacs "26.1") (dash "2.14.1") (dash-functional "2.14.1") (f "0.20.0") (ht "2.0") (spinner "1.7.3") (markdown-mode "2.3") (lv "0"))
-;; Version: 7.1.0
```

Comment:
2 options:

1. Revert this as melpa is pretty much disregarding this version and you can skip that step.
2. Do proper release and update the changelog and release the other components in emacs-lsp as well.

Reply:
1. Is changing the Version here too bad though? I think it is good style to be consistent.
2. I'll make a changelog entry. What other components are you referring to, though? I have made a PR for `lsp-treemacs`.

Reply:
> 1. Is changing the Version here too bad though? I think it is good style to be consistent.

Indeed - when you release one component you have to release the other. If you release 7.1 some of the other packages(lsp-java, lsp-metals, etc) has to be release as well.

> I'll make a changelog entry.

You have to make changelog entry for all changes since last release.

Reply:
But why release all others? `lsp-java` has absolutely nothing to do with it (I haven't touched it for this PR). Also, don't you think a release for this small function is overkill? Perhaps we can release this together with the one to remove `lsp-execute-command`.

Reply:
When you release lsp-mode it will be published in melpa stable where we have an old version of other components and there is a potential for unmatching components because during 7.1 release we have performed breaking changes(potentially).

Also, this is not the way to do release - first, you have to tag a version and then bump the version otherwise melpa stable won't have 7.1 version.


> Also, don't you think a release for this small function is overkill? 

I am not saying that you should do a release.

Reply:
I'll undo the release. I think its overkill here.